types/lumino-js-sdk.d.ts

```diff
@@ -182,15 +176,7 @@ declare module 'lumino-js-sdk' {
          *
          * @returns {Promise}
          */
-        makePayment({
-            amountOnWei,
-            tokenAddress,
-            partnerAddress
-        }: {
-            amountOnWei: number;
-            tokenAddress: string;
-            partnerAddress: string;
-        }): Promise<any>;
+        makePayment(payment: BalanceIdentifier): Promise<any>;
```

Comment:
And here doesn't make any sense to me to have `Identifier`, payment should be or payment or PaymentRequest if you want